### DISCUSSION TITLE
need help booting opi5 of nvme after spi update - problem

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#964] [Q&A]
so the other day i updated using
sudo u-boot-install-mtd /dev/mtdblock0
i was on sd at that moment and all was good, today nvme drive came by and it doesnt boot of it whatever i try so i guess i am missing something

i tried to dd sdcard on nvme
i tried to cat and i tried to dd downloaded image directly onto nvme
all these do without errors but it doesnt boot. i know its not the drive bcos i have older nvme with smarthome backup with armbian on that booted fine but doesnt boot any more. 
so for the end i tried dd rkspi_loader.img onto mtdblock0 and like before it finished fine but still no boot of nvme
so basicaly exactly what i wanted to avoid happened :/ and now need help to fix it.
what are my options? 
thanks for help!

drive is 2tb sn580, it will be changed for something slower for opi5 later but for now it is what it is if that helps
os is 24.04

edit:  also tried balena etcher to flash sd to nvme, no go

another edit:
managed to boot of nvme flashing ubot from here https://github.com/si0ls/u-boot-orangepi5, so it works atm but not sure if this is gonna work fine with jr ubuntu

**Comment by Joshua-Riek** [comment]
There is little I can do without U-Boot logs, in which you would need to connect a serial console.

**Reply by unevoljitelj** [reply]
Late with reply, yeah that uboot from the link works perfectly with your os. Dunno why it failed with yours, this other worked first try.
